src/web/mjs/connectors/Cycomi.mjs

```diff
@@ -8,54 +8,123 @@ export default class CyComi extends Connector {
         super.id = 'cycomi';
         super.label = 'CyComi';
         this.tags = ['manga', 'japanese'];
-        this.url = 'https://cycomi.com';
+        this.url = 'https://web.cycomi.com';
     }
 
     async _getMangaFromURI(uri) {
         const request = new Request(uri, this.requestOptions);
-        const id = uri.pathname;
-        const title = (await this.fetchDOM(request, 'div.title-texts > h3'))[0].textContent.trim();
+        const id = uri.pathname.split("/").pop();
+        const title = (await this.fetchDOM(request, 'h2.css-m40re5'))[0].textContent.trim();
         return new Manga(this, id, title);
     }
 
     async _getMangas() {
+        const uri = new URL('/api/title/serialization/list', this.url);
+        const request = new Request(uri, this.requestOptions);
+        const response = await this.fetchJSON(request, 'div.card-content > a.card');
         const mangaList = [];
-        for(let page = 0; page < 8; page++) {
-            const mangas = await this._getMangasFromPage(page);
+        for(let page = 0; page < 8; page += 1) {
```

Comment:
Use `response.data.serialization.length` instead of `8` which is likely to change in the near fututre